src/patternfly/components/DataList/data-list-item-actions.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="pf-c-data-list__item-actions{{#if data-list-item-actions--modifier}} {{data-list-item-actions--modifier}}{{/if}}"
```

Comment:
let's rename this to `pf-c-data-list__item-action` to be consistent with other components.